### PR TITLE
Create vumacc.json

### PR DESCRIPTION
**Pull request by Vumacc**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://vumack.web.app/

### CODE REVIEW

**Review by iostpa — APPROVED** [review]
Seems good to me (also if the domain doesn't seem to redirect I'd recommend you use [redirect.pizza](https://redirect.pizza))

File: domains/vumacc.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My terminal-styled portfolio website"
```
Comment (CuteDog5695) [inline domains/vumacc.json:2]:
```suggestion
  "description": "My terminal-styled portfolio website",
```

**Review by iostpa — CHANGES_REQUESTED** [review]
Oh, it was invalid

**Review by iostpa — APPROVED** [review]
Now its fine

**Pull request merged**